### ISSUE TITLE
Custom SRID edit error

### ISSUE THREAD

**Issue by marcing** [phpmyadmin/phpmyadmin#18527]
### Describe the bug

For column types with custom SRID it's not possible to edit the row value, as of version phpMyAdmin 5.2.1, MySQL version 8.0.33.

### To Reproduce

Import database:

```
CREATE TABLE IF NOT EXISTS `locations` (
  `id` BIGINT UNSIGNED NOT NULL AUTO_INCREMENT,
  `name` VARCHAR(256) NOT NULL,
  `latitude` DECIMAL(10,8) NOT NULL,
  `longitude` DECIMAL(11,8) NOT NULL,
  `location` POINT SRID 4326 NOT NULL,
  PRIMARY KEY (`id`),
  UNIQUE INDEX `id_UNIQUE` (`id` ASC) VISIBLE,
  FULLTEXT INDEX `name` (`name`) VISIBLE,
  INDEX `latitude_longitude` (`latitude` ASC, `longitude` ASC) INVISIBLE,
  SPATIAL INDEX `location` (`location`) INVISIBLE)
ENGINE = InnoDB;

INSERT INTO `locations` (`id`, `name`, `latitude`, `longitude`, `location`) VALUES
(1, 'Eisenstadt (7000)', 47.84440844, 16.52665203, ST_GeomFromText(CONCAT('POINT(', latitude, ' ', longitude, ')'), 4326));

```

Browse the table, click Edit on first row, click Go to save the row (screenshot with error attached).

![obraz](https://github.com/phpmyadmin/phpmyadmin/assets/190209/363323bd-21e9-4e9a-a5ad-2843d7b297d1)

**Comment by williamdes** [comment]
Probably related to #18513

@MoonE you did edit quite some gis stuff recently. Do you think it's doable in QA?

**Comment by MoonE** [comment]
I think this was already fixed by #18188

**Comment by williamdes** [comment]
Thanks @MoonE
@marcing cn you try the latest 5.2 version in development [(phpMyAdmin 5.2+snapshot)](https://www.phpmyadmin.net/downloads/#snapshot_5.2+snapshot)?

**Comment by marcing** [comment]
@williamdes @MoonE  I'm sorry it took so long, but yes I confirm the snapshot has a proper fix! Thank you very much!

**Comment by williamdes** [comment]
Fixed by #18188

Thank you for the feedback @marcing 🎉
